sh_scrapy/crawl.py

```diff
@@ -78,7 +78,8 @@ def _fatalerror():
 
 def _get_apisettings():
     from sh_scrapy.env import decode_uri
-    return decode_uri(envvar='JOB_SETTINGS') or {}
+    return (decode_uri(envvar='SHUB_SETTINGS') or
```

Comment:
I think it makes sense to stick to new env vars only. New version of scrapinghub-entrypoint-scrapy will rely on new vars, old will work with old vars because we keep backwards compatibility.